content/efcore/getting-started/samples/defining-entity-relationships/one-to-one-manual-foreign-key/LibraryContext.cs

```diff
@@ -12,7 +12,12 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         modelBuilder.Entity<Reader>()  
             .HasOne(r => r.Address) 
             .WithOne(a => a.Reader)  
-            .HasForeignKey<Address>(a => a.ReaderId); 
+            .HasForeignKey<Address>(a => a.ReaderId);
+
+        modelBuilder.Entity<Address>()
```

Comment:
Don't need this. We only need to specify `ReaderId` above because we are specifying which one has the foreign key in the database.